.github/workflows/release_sdk.yml

```diff
@@ -0,0 +1,45 @@
+name: Release Selfcare sdk
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - './libs/**'
+
+  workflow_dispatch:
+
+jobs:
+  setup:
+    name: Release
+    runs-on: ubuntu-24.04
+    permissions:
+      packages: write
+
+    steps:
+
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # 4.1.1
+        with:
+          ref: ${{ github.ref_name }}
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@0ab4596768b603586c0de567f2430c30f5b0d2b0 # pin@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          cache: maven
+
+      - uses: s4u/maven-settings-action@60912582505985be4cc55d2b890eb32767f8de5f # pin@v2.8.0
+        with:
+          servers: '[{"id": "selfcare-github", "username": "${{ github.actor }}", "password": "${{ secrets.GITHUB_TOKEN }}"}]'
+
+      - name: Build with Maven
+        shell: bash
+        run: mvn --projects :selfcare-libs --also-make-dependents clean package -DskipTests
+
+      - name: Deploy to GitHub Package Registry
+        shell: bash
+        run: |
+          mvn -B deploy -f libs/pom.xml
```

Comment:
If i've understood correctly, with this structure, we are releasing all libs in one whit this command right? If so, i wish separate each lib with its own version (read on pom artifact version)